docker-compose.yml

```
@@ -27,3 +27,13 @@ services:
       - '1323:1323'
     tty: true
     stdin_open: true
+  view:
+    build: ./view
+    container_name: 'view'
+    ports:
+      - "3000:3000"
+    command: sh -c "npm install && npm run dev"
+    volumes:
+      - ./view:/app
+    stdin_open: true
+    tty: true
```

```suggestion
    tty: true
    
```
プログラミングは最終行は空行で終わることが望ましいです！
おそらくvscodeを利用していると思うのですが、vscodeは自動で最終行を改行してくれるので設定しちゃいましょう！
以下の記事が参考になります。
https://qiita.com/kyo_nanba/items/d384b2a46170bf8764dd